sbirez/management/commands/importtopics.py

```diff
@@ -27,15 +27,15 @@ def handle(self, mdb_filename, solicitation_name, clear=False, **options):
             raise OSError("MS Access database %s not found" % mdb_filename)
 
         # Confirm that solicitation_name is for solicitation in the database
-        proc = subprocess.Popen(#'ls',
```

Comment:
@catherinedevlin: I re-added the check, but added a new check to not throw a CommandError if the shell out can't find the `psql` command.